libs/app-routing/feature/src/lib/legal-layout/layout.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { FunctionComponent } from "react"
+import { Outlet } from "react-router"
+import styled from "styled-components"
+
+export const LegalLayout: FunctionComponent = () => {
+  return (
+    <Wrapper>
+      <Outlet />
+    </Wrapper>
+  )
+}
+
+const Wrapper = styled.div`
+  height: 100vh;
+  overflow-y: auto;
+  box-sizing: border-box;
```

Comment:
I'd only add some horizontal padding here, to avoid issues like this 

<img width="573" alt="image" src="https://github.com/user-attachments/assets/6ce8ab4c-18be-4c7c-9ba8-65f7e57b8d2b" />